views/js/previewer/plugins/button.tpl

```diff
@@ -0,0 +1,19 @@
+<li
+    data-control="{{control}}"
+    class="small btn-info action{{#if className}} {{className}}{{/if}}"
+    title="{{title}}"
```

Comment:
If the title is here only for screen readers, it should be changed to smth that is supported better. Could be a `visually-hidden` technique (best support) or `aria-label`